### PR TITLE
Add 22

### PR DESCRIPTION
**Pull request by elifaslan1**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: bin/update-dockerfiles.sh [hunk 1]
```diff
@@ -104,6 +104,7 @@ CORRETTO_8_GENERIC_LINUX=$(cat versions.json | jq -r '.["8"]' )
 CORRETTO_11_GENERIC_LINUX=$(cat versions.json | jq -r '.["11"]' )
 CORRETTO_17_GENERIC_LINUX=$(cat versions.json | jq -r '.["17"]' )
 CORRETTO_21_GENERIC_LINUX=$(cat versions.json | jq -r '.["21"]' )
+CORRETTO_22_GENERIC_LINUX=$(cat versions.json | jq -r '.["22"]' )
```
Comment (lutkerd) [inline bin/update-dockerfiles.sh:107]: We should discuss this and potentially not release a docker for AL2 w/22.

File: 22/jdk/alpine/3.19/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.19
+
+ARG version=22.0.0.36-2
```
Comment (lutkerd) [inline 22/jdk/alpine/3.19/Dockerfile:3]: The version here should be 22.0.0.36.2, not sure why this is different from previous versions yet.

File: bin/tag-generator.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 DEFAULT_ALPINE_VERSION = '3.19'
 ALPINE_VERSIONS = ['3.16', '3.17', '3.18', '3.19']
 
-LTS_VERSIONS = [ "8", "11", "17", "21"]
+LTS_VERSIONS = [ "8", "11", "17", "21", "22"]
```
Comment (lutkerd) [inline bin/tag-generator.py:6]: 22 is not LTS and should be here.

File: bin/update-dockerfiles.sh [hunk 4]
```diff
@@ -4,7 +4,7 @@ SED="sed -i"
 
 sed --version 2>/dev/null || SED="sed -i.bkp"
 
-LTS_VERSIONS=("8" "11" "17" "21")
+LTS_VERSIONS=("8" "11" "17" "21" "22")
```
Comment (lutkerd) [inline bin/update-dockerfiles.sh:7]: 22 is not LTS and should be here.

**Comment by max-grigoriev** [comment]
The documentation already says that JDK 22 is available: https://docs.aws.amazon.com/corretto/latest/corretto-22-ug/docker-install.html#alpine-images which isn't true :(

**Comment by Hc747** [comment]
Bump - is there anything blocking the release of JDK 22 images?

**Review by lutkerd — APPROVED** [review]
Please squash when committing.

**Pull request merged**